Source/WebCore/html/HTMLSummaryElement.cpp

```diff
@@ -94,7 +94,8 @@ void HTMLSummaryElement::defaultEventHandler(Event& event)
 {
     if (isActiveSummary()) {
         auto& eventNames = WebCore::eventNames();
-        if (event.type() == eventNames.DOMActivateEvent && !isInSummaryInteractiveContent(event.target())) {
+        RefPtr target = event.target();
```

Comment:
We don't need this local variable, we should call `protectedTarget()` below instead.